common/src/main/java/com/microsoft/identity/common/nativeauth/internal/commands/AcquireTokenNoFixedScopesCommand.kt

```diff
@@ -47,20 +45,9 @@ class AcquireTokenNoFixedScopesCommand(
     }
 
     override fun execute(): AcquireTokenResult {
-        LogSession.logMethodCall(
-            tag = TAG,
-            correlationId = parameters.getCorrelationId(),
-            methodName = "${TAG}.execute"
-        )
-
         val result = controller.acquireTokenSilent(
             parameters
         )
-
-        Logger.info(
```

Comment:
Is the result logged in MSAL's SDK interface (when receiving the result) or in the Controller (when returning the result)? In my opinion this is valuable information. it's the summary of all the internal processing the SDK does.